test_runner/src/main/kotlin/ftl/cli/firebase/test/CommonRunCommand.kt

```diff
@@ -199,4 +199,17 @@ abstract class CommonRunCommand {
                 "which don't support ansi codes, to avoid corrupted output use `single` or `verbose`."]
     )
     var outputStyle: String? = null
+
+
+    @CommandLine.Option(
+        names = ["--ci-junit-result"],
```

Comment:
I think since the default use case is CI, we probably want to generate this file by default. I'm not sure we need to enable the filename to be customized, we're not providing that option for other generated files.

Reply:
Alright, I will change it.